semanticdb/semanticdb3/semanticdb3.md

```diff
@@ -2377,8 +2377,8 @@ In this section, we describe the Java symbol format.
 
 **Descriptor** is:
   * For `LOCAL`, unsupported.
-  * For `FIELD` or `PACKAGE`, concatenation of its simple name [\[98\]][98]
-    and a dot (`.`).
+  * For `PACKAGE`, concatenation of its encoded name and a forward slash (`/`).
```

Comment:
It doesn't look like the Java part of the spec defines what's an encoded name.

Reply:
Done.